src/prefect/events/clients.py

```diff
@@ -349,15 +349,21 @@ async def __aexit__(
         return await super().__aexit__(exc_type, exc_val, exc_tb)
 
     async def _reconnect(self) -> None:
+        logger.debug("Reconnecting websocket connection.")
```

Comment:
any upside to using some reference ID in the logs, e.g., `id(self)`? Not requesting you make any changes, just tossing it out in case it's useful

Reply:
good call, added!